OpenRA.Mods.Common/Traits/Render/WithDamageOverlay.cs

```diff
@@ -16,7 +16,7 @@
 namespace OpenRA.Mods.Common.Traits.Render
 {
 	[Desc("Renders an overlay when the actor is taking heavy damage.")]
-	public class WithDamageOverlayInfo : TraitInfo, Requires<RenderSpritesInfo>
+	public class WithDamageOverlayInfo : TraitInfo, Requires<RenderSpritesInfo>, Requires<BodyOrientationInfo>
```

Comment:
shouldn't body orientation be optional?

Reply:
We can make it optional, but that trait does not make sense without it. (All our actors have body orientation.) `WithIdleOverlay` also requires it.

Reply:
Adjusted it.